### PR TITLE
Solution

### PR DESCRIPTION
**Pull request by MarinaT-N**

### CODE REVIEW

File: cypress.config.js [hunk 1]
```diff
@@ -1,8 +1,11 @@
-const { defineConfig } = require('cypress')
+const { defineConfig } = require("cypress");
 
 module.exports = defineConfig({
-  e2e: {
-    setupNodeEvents(on, config) {
-    },
+  e2e: {   
+           
+        baseUrl: "http://localhost:1667",
```
Comment (NataliiaSolotva) [inline cypress.config.js:6]: Next time test the app you are asked for, please (https://react-redux.realworld.io)

**Review by NataliiaSolotva — COMMENTED** [review]
Gooooood job!

**Review by NataliiaSolotva — APPROVED** [review]
Gooooood job!

File: cypress.config.js (hunk 1 above)
Comment (MarinaT-N) [reply cypress.config.js:6]: Ahhhhh..... i even didn't notice it.... i will also do the correct one . thanks

**Pull request closed without merging**